src/components/basic/TextWithTooltip.md

```diff
@@ -29,6 +29,22 @@ import TextWithTooltip from '../basic/TextWithTooltip.jsx';
 </Container>;
 ```
 
+### Change TextWithTooltip's trigger delay
+```jsx
+import { Container } from '@zextras/carbonio-design-system';
+import TextWithTooltip from '../basic/TextWithTooltip.jsx';
+<Container orientation="horizontal" mainAlignment="space-around" height={100}>
+	<TextWithTooltip
+        label="3 seconds before tooltip trigger Lorem ipsum dolor sit amet"
+        triggerDelay={3000}
+        style={{ maxWidth: '100px' }}
```

Comment:
This might prove the loss of styling: here the style prop is applied to the Tooltip component, so the Text have auto width (and is fully visible, tooltip is not even triggered since text is not ellipsed), and instead the width is applied to the Tooltip itself.